denonavr/ssdp.py

```diff
@@ -13,51 +13,51 @@
 import xml.etree.ElementTree as ET
 from urllib.parse import urlparse
 import requests
+import ifaddr
```

Comment:
This is a 3rd party package, so it should be included into setup.py

Reply:
thanks!
I was already worried about this, I did not know how to make sure someone has this module installed.

Has now been fixed